src/types/redis_hash.h

```diff
@@ -29,13 +29,23 @@
 #include "storage/redis_db.h"
 #include "storage/redis_metadata.h"
 
-typedef struct FieldValue {
+struct FieldValue {
   std::string field;
   std::string value;
-} FieldValue;
+};
 
 enum class HashFetchType { kAll = 0, kOnlyKey = 1, kOnlyValue = 2 };
 
+struct HashSpec {
```

Comment:
I think that it should be `HashRangeSpec` or something like that to describe that this is a spec for a range command in a hash.